protos/sources/vega/governance.proto

```diff
@@ -352,6 +352,76 @@ message ProposalTerms {
   }
 }
 
+// Terms change for a batch governance proposal
```

Comment:
Shouldn't these governance.proto changes also be reflected in GraphQL?

Reply:
There is another ticket for the API. It will be reflected there